src/popup/components/TransferSend/TransferSendRecipient.vue

```diff
@@ -85,9 +85,10 @@ export default defineComponent({
   setup(props) {
     const { openModal } = useModals();
     const { activeAccount } = useAccounts();
-    const { getTippingUrlStatus } = useAeTippingUrls();
+    const isAeternity = computed(() => props.protocol === PROTOCOL_AETERNITY);
+    const { getTippingUrlStatus } = useAeTippingUrls(isAeternity.value);
 
-    const urlStatus = computed(() => getTippingUrlStatus(props.modelValue));
+    const urlStatus = computed(() => isAeternity.value ? getTippingUrlStatus(props.modelValue) : 'not-verified');
```

Comment:
If you are using string values please force the return type to validate it:
```suggestion
    const urlStatus = computed((): UrlStatus => isAeternity.value ? getTippingUrlStatus(props.modelValue) : 'not-verified');
```

Reply:
Absolutely correct, I wasn't aware this is possible with computed properties